core/vm/evm.go

```diff
@@ -400,6 +483,7 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 		}
 	}
 
+	// CS TODO: decide what to do with this condition. Should we bring it top or add firehose operation on this
```

Comment:
We probably need to end the call before returning from the function inside `if`.